includes/ucf-news-config.php

```diff
@@ -12,8 +12,27 @@ class UCF_News_Config {
 				'topics'    => '',
 				'limit'     => 3,
 				'per_row'   => 3
+			),
+			$default_plugin_options = array(
+				'ucf_news_feed_url'       => 'http://today.ucf.edu/wp-json/wp/v2/posts/',
+				'ucf_news_include_css'    => 'on',
+				'ucf_news_fallback_image' => ''
 			);
 
+		public static function add_options() {
+			$defaults = self::$default_plugin_options;
+
+			add_option( 'ucf_news_feed_url', $defaults['ucf_news_feed_url'] );
+			add_option( 'ucf_news_include_css', $defaults['ucf_news_include_css'] );
+			add_option( 'ucf_news_fallback_image', $defaults['ucf_news_include_css'] );
```

Comment:
The option value on line 27 needs to be updated from `$defaults['ucf_news_include_css']` to `$defaults['ucf_news_fallback_image']`.